content/en/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md

```diff
@@ -0,0 +1,164 @@
+You can configure an ingress gateway for the cluster-local Prometheus, providing external connectivity to the in-cluster
+Prometheus endpoint.
+
+For each cluster, follow the appropriate instructions from the [Remotely Accessing Telemetry Addons](/docs/tasks/observability/gateways/#option-1-secure-access-https) task. And you
+**SHOULD** establish secure (HTTPS) access.
+
+After that, you will need to configure your external Prometheus instance to access the cluster-local Prometheus instances.
+This can be achieved with a configuration like the following (replacing the gateway address and cluster name):
+
+{{< text yaml >}}
+scrape_configs:
+  - job_name: 'federate-{{CLUSTER_NAME}}'
+    scrape_interval: 15s
+
+    honor_labels: true
+    metrics_path: '/federate'
+
+    params:
+      'match[]':
+        - '{job="pilot"}'
+        - '{job="envoy-stats"}'
+
+    static_configs:
+      - targets:
+        - '{{GATEWAY_IP_ADDR}}:15030'
+        labels:
+          cluster: {{CLUSTER_NAME}}
+{{< /text >}}
+
+Notes:
+
+* `CLUSTER_NAME` should be set to the same value which is used to create the cluster (set via `values.global.multiCluster.clusterName`).
+
+* No authentication to the Prometheus endpoint(s) is provided. This means that anyone can query your
+cluster-local Prometheus instances. This may not be desirable.
+
+* Without proper HTTPS configuration of the gateway, everything is being transported via plaintext. This may not be
+desirable.
+
+### Production Prometheus from one of the clusters
+
+If you desire to run a Prometheus in one of the clusters, establish connectivity from the production instance of
+Prometheus to each of the cluster-local Prometheus instances within the mesh.
+
+This is really just a customization of the process for external federation, which you can achieve by configuring the `Gateway`,
+`VirtualService`, and `DestinationRule` in each remote cluster.
+
+{{< image width="80%"
+    link="./in-mesh-production-prometheus.svg"
+    alt="Architecture of in-mesh Production Prometheus for monitoring multicluster Istio."
+    caption="In-mesh Production Prometheus for monitoring multicluster Istio"
+    >}}
+
+Configure your production Prometheus to access both of the *local* and the *remote* Prometheus instances. This can be achieved
+by adding a configuration like the following for the *remote* clusters (replacing the service name and cluster name for each cluster):
+
+{{< text yaml >}}
+scrape_configs:
+  - job_name: 'federate-{{CLUSTER_NAME}}'
+    scrape_interval: 15s
+
+    honor_labels: true
+    metrics_path: '/federate'
+
+    params:
+      'match[]':
+        - '{job="pilot"}'
+        - '{job="envoy-stats"}'
+
+    static_configs:
+      - targets:
+        - '{{GATEWAY_IP_ADDR}}:15030'
+        labels:
+          cluster: {{CLUSTER_NAME}}
+{{< /text >}}
+
+Then add a configuration like the following for the *local* cluster:
+
+{{< text yaml >}}
+- job_name: 'federate-local'
```

Comment:
Why is ther no `scrape_config:` and proper formatting (indent and blank lines) in this example, like the previous ones?

Reply:
- since I think according to the context, we are assuming to add this job to the above scrape_config? If we add scrape_config here, it would be causing confusion I guess. 
- Yes, indentation needed. Let me fix it!

Reply:
If so, then that is totally unclear from the above sentence where it just says `add a configuration like the following ...`.

You would need to say something like `append the following config to the above scrape_config and then apply it in the *local* cluster:`. Or, you could show the full config with both, which would be easier for the user to cut and paste. Either way, my formatting question still applies. Why is it not indented 2 spaces, so it's correct when the user cut/pastes it? Also why not include the blank lines, for readability?

As an aside, most docs give the full command that the user can cut and paste, instead of just the YAML. Why are you not doing that here?

Reply:
Then, I think combine the two jobs to `scrape_config` to provide the users with a fully yaml would be the idea way to follow other docs' conventions. Let me gather them together.